amplify-tools-gradle-plugin/src/main/groovy/com/amplifyframework/tools/gradle/plugin/amplifytools.groovy

```diff
@@ -0,0 +1,226 @@
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+import groovy.json.JsonSlurper
+import groovy.json.JsonOutput
+
+class amplifytools implements Plugin<Project> {
+    void apply(Project project) {
+        def doesNodeExist = true
```

Comment:
Because I keep getting some emails related to this PR, what is the difference between this line and line 16

Reply:
Hey Sam, line 16 has to do with toggling datastoreSync. If you mean the verifyNode task, that task sets ```doesNodeExist``` to false if node isn't found so that other tasks requiring node don't get executed (using the variable since a task cannot execute another task in gradle). An alternative which John mentioned is just erroring out of the build in the verifyNode task since the other tasks depend on it. Currently trying that out